plugins/pipelines/db/migrate/20180510172630_add_pipelined_to_deploys.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPipelinedToDeploys < ActiveRecord::Migration[5.1]
+  def change
+    add_reference :deploys, :triggering_deploy, foreign_key: {to_table: :deploys}, type: :integer
```

Comment:
we don't use the index since we don't show which deploys were triggered

Reply:
generally prefer the more sql-style migrations since it makes it more obvious what will be done ... but ok either way